src/vanilla/Templates/ModelTemplate.cshtml

```diff
@@ -4,175 +4,171 @@
 @using AutoRest.Core.Model
 @using AutoRest.Python
 @using AutoRest.Python.Model
-@inherits AutoRest.Python.PythonTemplate<AutoRest.Python.Model.CompositeTypePy>
+@inherits AutoRest.Python.PythonTemplate<System.Collections.Generic.List<AutoRest.Python.Model.CompositeTypePy>>
 # coding=utf-8
 # --------------------------------------------------------------------------
 @Header("# ").TrimMultilineHeader()
 # --------------------------------------------------------------------------
 @EmptyLine
-@if(Model.ComplexConstants.Any())
```

Comment:
I'm a little nervous to see this disapearing totally from this file. Do you have any reasons to believe it was unecessary?